HLTrigger/Configuration/python/customizeHLTforPatatrack.py

```diff
@@ -0,0 +1,485 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I would *think* the content of this file should be moved to `HLTrigger/Configuration/python/customizeHLTforAlpaka.py` once https://github.com/cms-sw/cmssw/pull/43257 is merged?

Reply:
Indeed, that would be the right place.